.github/workflows/build-and-deploy.yml

```diff
@@ -112,6 +113,31 @@ jobs:
           environment: development_aks
           azure_credentials: ${{ secrets.AZURE_CREDENTIALS }}
 
+  deploy_v2_test:
+    name: Deploy to test_aks environment
+    concurrency: deploy_v2_test
+    needs: [docker]
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+    environment:
+      name: test_aks
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/workflows/actions/deploy_v2
+        id: deploy_v2
+        with:
+          environment: test_aks
+          docker_image: ${{ needs.docker.outputs.docker_image }}
```

Comment:
the docker_image isn't getting through to the k8 app & worker, need to check where this is going wrong. And should remove default 2ad42e8958a4d63ed295a58a0847430705725ba8 from the Makefile

Reply:
Removed the default in Makefile - however I can see the image in K8?